pythran/pythonic/__builtin__/abs_impl.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef PYTHONIC_BUILTIN_ABS_IMPL_HPP
+#define PYTHONIC_BUILTIN_ABS_IMPL_HPP
+
+#include "pythonic/utils/proxy.hpp"
+#include "pythonic/__builtin__/abs.hpp"
+
+#include <nt2/include/functions/abs.hpp>
+
+namespace pythonic {
+
+    namespace __builtin__ {
+
+        ALIAS(abs, nt2::abs);
+        PROXY_IMPL(pythonic::__builtin__, abs);
+
+    }
+
+}
+
+#endif
```

Comment:
As all others, it is a separation between implementation and declaration. Declaration is done in pythran/pythonic/__builtin__/abs.hpp